explorer-ui/src/components/LoginForm/RegisterForm/RegisterForm.jsx

```diff
@@ -0,0 +1,168 @@
+import * as React from "react"
+import "./RegisterForm.css"
+import axios from "axios"
+import * as config from "../../../config"
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+import Footer from "../../Home/Footer/Footer";
+import NavBar from "../../Home/NavBar/NavBar";
+export default function RegisterForm({ handleLogin }) {
+    const username = React.createRef();
+    const password = React.createRef();
+    const age = React.createRef();
+    const email = React.createRef();
```

Comment:
would store these in state instead as refs generally for DOM manipulation / are limiting later as components scale up and need to do more complex logic

Reply:
I tried using states by setting the values of the input field to be the state's value and then creating an "onChange" function that sets the state to the value of that event but it was causing a bug where after typing in a single character the screen would momentarily freeze.